### PR TITLE
[Build] change dir structure - @open sesame 01/11 14:00

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. update meson/cmake to change directory structure (move common files and plugins to gst/nnstreamer)
2. move nnstreamer meson-build to gst/nnstreamer
3. set project args to build nnstreamer
4. change option to avoid duplicated feature name

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

**Comment by myungjoo** [comment]
> remove warning in testcases

Is this warning caused by incorrect path?

**Comment by jaeyun-jung** [comment]
@myungjoo LD_PRELOAD is unnecessary when adding gst-plugin-path in pipeline.

File: CMakeLists.txt [hunk 1]
```diff
@@ -63,7 +63,7 @@ ENDIF(ENABLE_ORC)
 pkg_check_modules(pkgs REQUIRED ${PKG_MODULES})
 
 INCLUDE_DIRECTORIES(
-	${CMAKE_SOURCE_DIR}/gst/nnstreamer
+	${CMAKE_SOURCE_DIR}/gst/nnstreamer/common
```
Comment (myungjoo) [inline CMakeLists.txt]: Let's keep the common files at /gst/nnstreamer/
We do not need "common" subdirectory

File: CMakeLists.txt [hunk 2]
```diff
@@ -104,7 +104,7 @@ ENDIF(ENABLE_TEST)
 
 SET(OBJECTS)
 FOREACH(PRJ IN LISTS PROJECTS)
-	ADD_SUBDIRECTORY(gst/${PRJ})
+	ADD_SUBDIRECTORY(gst/nnstreamer/${PRJ})
```
Comment (myungjoo) [inline CMakeLists.txt]: This is good.

File: CMakeLists.txt (hunk 1 above)
Comment (jaeyun-jung) [reply CMakeLists.txt]: @myungjoo updated to remove common folder

**Comment by myungjoo** [comment]
@helloahn @wooksong @jijoongmoon : You guys are supposed to review commits. A lot of reviewers are out of office. We are not going to enforce this, but let's try to have two or more approvals before merging.

**Pull request merged**